src/Avalonia.Animation/Animation.cs

```diff
@@ -194,6 +194,33 @@ public string RepeatCount
         [Content]
         public KeyFrames Children { get; } = new KeyFrames();
 
+        // Store values for the Animator attached properties.
+        private static readonly Dictionary<object, Type> s_animators = new Dictionary<object, Type>();
+
+        /// <summary>
+        /// Gets the value of the Animator attached property for an object.
+        /// </summary>
+        /// <param name="obj">The object.</param>
+        /// <returns>The property animator type.</returns>
+        public static Type GetAnimator(object obj)
```

Comment:
I wonder, can "object" be of specific type? IAnimationSetter

Reply:
I think it can be yeah

Reply:
Changed type